### ISSUE TITLE
Regarding AIR Loss

### ISSUE THREAD

**Issue by HashmatShadab** [GodXuxilie/Enhancing_ACL_via_AIR#1]
https://github.com/GodXuxilie/Enhancing_ACL_via_AIR/blob/64252c8036357d550bc9b960f1959843d268c4be/AIR.py#L96

From the above line, the logits will not remain a probability distribution. Isn't there need to apply softmax before using the KL divergenece loss?
